src/lwc/psElevateTokenHandler/psElevateTokenHandler.js

```diff
@@ -0,0 +1,168 @@
+
+import { fireEvent } from 'c/pubsubNoPageRef';
+import { getNamespace, isFunction, isNull } from 'c/utilCommon';
+
+import PAYMENT_AUTHORIZATION_TOKEN_FIELD from '@salesforce/schema/DataImport__c.Payment_Authorization_Token__c';
+import tokenRequestTimedOut from '@salesforce/label/c.gePaymentRequestTimedOut';
+
+
+/***
+* @description Visualforce page used to handle the payment services credit card tokenization request
+*/
+const TOKENIZE_CARD_PAGE_NAME = 'GE_TokenizeCard';
+
+/***
+* @description Event action sent to the Visualforce page to request the token
+*/
+const TOKENIZE_EVENT_ACTION = 'createToken';
+
+/***
+* @description Max number of ms to wait for the response containing a token or an error
+*/
+const TOKENIZE_TIMEOUT_MS = 10000;
+
+
+/***
+* @description Payment services Elevate credit card tokenization service
+*/
+class psElevateTokenHandler {
```

Comment:
@ptewson-sfdo , a new Elevate token handler was created for the common methods. I have not changed the GE form, but in the case the Visualforce origin handling is changed and this PR is not merged into the `master` yet, feel free to copy the class and use/update the common methods.